src/components/Tabs.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { Tab } from './Tab';
+
+export class Tabs extends PureComponent {
+  state = {
+    tabContent: this.props.tabs[this.props.index].content,
+    currTabId: 0,
+  }
+
+  onTabSelected = (content, index) => {
+    this.setState({
+      tabContent: content,
+      currTabId: index,
+    });
+  }
+
+  render() {
+    const { tabs } = this.props;
+    const { tabContent } = this.state;
+
+    return (
+      <div className="ui container">
+        <div className="ui top attached tabular menu">
+          {
+            tabs.map(({ title, content, index }) => (
+              <Tab
+                title={title}
+                onTabSelected={() => this.onTabSelected(content, index)}
+                key={index}
```

Comment:
it's a bad practice to use index as a key
```suggestion
                key={title}
```

Reply:
done